### ISSUE TITLE
ensure we're compatible with changes in graphics device

### ISSUE THREAD

**Issue by kevinushey** [rstudio/rstudio#7253]
See: https://github.com/wch/r-source/commit/b2a0049a74768dc8e38e984109829f4605f883af

Fortunately, this change only adds new members to the end of the `_DevDesc` struct (it doesn't change the layout of pre-existing members) so we should be safe. Still, worth validating this is indeed the case.

**Comment by jmcphers** [comment]
We will also need to bump our graphics engine support version as part of this change (see https://github.com/rstudio/rstudio/issues/7277)

**Comment by R-Hannibal** [comment]
Verified Not Fixed on: RStudio Version - 1.3.1058, R Version - R-devel (4.1.0 in development), Operating System - Windows 10 x64/Desktop 

Still seeing the same functional behavior when comparison testing between 1.3.1056 and 1.3.1058: plot is displayed in a pop-out, warning is displayed when entering `plot(1:10)` - 

```
Warning message:
R graphics engine version 13 is not supported by this version of RStudio. 
The Plots tab will be disabled until a newer version of RStudio is installed.
```

**Comment by kevinushey** [comment]
Looks like this is fixed in the latest daily builds (v1.4), and the associated patch _is_ on the v1.3 branch as well. So perhaps somehow the patch was merged after the preview was built? See: https://github.com/rstudio/rstudio/commit/5b32ae1247dfc35174c71f35362bbe4f5a558613

**Comment by kevinushey** [comment]
Ultimately I think this just needs to be re-tested and re-verified when the next preview release is built.

**Comment by R-Hannibal** [comment]
Verified Fixed on: RStudio Version - 1.3.1070, R Version - 4.1.0 (Under Development), Windows 10 x64 - Desktop

The warning is no longer displayed on the latest preview build and the plot is rendered/displayed correctly.

**Comment by kevinushey** [comment]
Thanks for catching that one @R-Hannibal! Sorry I let that slip through.
